commcare_connect/data_export/views.py

```
@@ -206,6 +215,41 @@ def get_queryset(self, request, opp_id):
         )
 
 
+class LabsRecordDataView(OpportunityDataExportView, ListCreateAPIView):
+    serializer_class = LabsRecordDataSerializer
+
+    def get_queryset(self, request, opp_id):
```

Ruff (0.14.4)

Unused method argument: `opp_id`

(ARG002)